### ISSUE TITLE
IDRID data set is divided into training set, test set and validation set

### ISSUE THREAD

**Issue by linyouth2022** [NKUhealong/PMCNet#5]
Dear author, how is the IDRID dataset divided to achieve the effect in the paper?
